src/pages/en/guides/content-collections.mdx

```diff
@@ -16,166 +16,196 @@ import TypeScriptSettingTabs from '~/components/tabs/TypeScriptSettingTabs.astro
 
-Astro generates TypeScript types from your content collections in the `.astro` directory. These types will be updated anytime you run the [`astro dev`](/en/reference/cli-reference/#astro-dev), [`astro build`](/en/reference/cli-reference/#astro-build), or [`astro sync`](/en/reference/cli-reference/#astro-sync) commands.
+The `.astro` directory will be updated for you automatically anytime you run the [`astro dev`](/en/reference/cli-reference/#astro-dev), [`astro build`](/en/reference/cli-reference/#astro-build) commands. You can run [`astro sync`](/en/reference/cli-reference/#astro-sync) at any time to update the `.astro` directory manually.
 
 :::tip
-If you're using Git for version control, we recommend ignoring this generated directory by adding `.astro` to your `.gitignore`.
-:::
-
-These commands will also generate a [reference path](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-path-) to include `.astro` types in your project. A [`src/env.d.ts` file](/en/guides/typescript/#setup) will be created for you if one does not exist, and the following will be added:
+If you're using Git for version control, we recommend ignoring the `.astro` directory by adding `.astro` to your `.gitignore`. This tells Git to ignore this directory and any files inside of it.
 
-```ts title="src/env.d.ts"
-// using a relative path `../` to the `.astro` directory
-/// <reference types="../.astro/types.d.ts" />
+```bash
+echo ".astro" >> .gitignore
 ```
+:::
 
-## The content directory
-
-Astro treats the `src/content/` directory as special. This is where **collections** (folders) of Markdown/MDX **entries** (files) can be stored, with a single configuration file to define each collection's **schema** (frontmatter data types and shape). Files other than your `.md`/`.mdx` content are not permitted inside `src/content/`.
 
-## Collections
 
-A collection is a directory in `src/content/` containing Markdown or MDX files. Every Markdown or MDX file in `src/content/` **must** belong to a collection directory, since Astro [provides built-in functions](#querying-content-collections) for querying your content by the collection directory name. 
+### Organizing with multiple collections 
 
-Content within a collection should share the same frontmatter shape and types. You can optionally enforce these types [by configuring a schema](/en/guides/content-collections/#defining-a-collection-schema).
+If two files represent two different kinds of content (e.g. a blog post and an author profile), they most likely belong in two different collections. This is an important concept to remember because important features like frontmatter validation and automatic TypeScript typings require that all entries share a similar frontmatter structure.
```

Comment:
```suggestion
If two files represent different kinds of content (e.g. a blog post and an author profile), they most likely belong in different collections. Helpful features like frontmatter validation and automatic TypeScript typings require that all entries in the same collection share a similar frontmatter structure.
```
I came for the double "important" and I stayed to avoid narrating the remembering process.

I chose "helpful" features, but could also see "powerful" features. I think those are more helpful to the reader than "important" (since these are not actually required).